packages/sdk/lib/deps.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
how about naming these files `dependencies.json` and `devDependencies.json` so they correspond to package.json equivalents? less things to learn/remember than having an abbreviation